### PR TITLE
remove nerpa reference since its deprecated

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build failed...

```

> filecoin-docs@1.0.0 docs:build /home/runner/work/filecoin-docs/filecoin-docs
> npm run docs:links && vuepress build docs


> filecoin-docs@1.0.0 docs:links /home/runner/work/filecoin-docs/filecoin-docs
> vuepress check-md docs


Checking markdown...

3 dead links was found

  Hash is not found: [Storage/Deals status](#storagedeals-status) (/home/runner/work/filecoin-docs/filecoin-docs/docs/reference/README.md:19:7)
  Hash should slugify: [APIs & Developer tools](#apis--developer-tools) (/home/runner/work/filecoin-docs/filecoin-docs/docs/reference/README.md:21:5)
  Hash should slugify: [Node Infrastructure & APIs](#node-infrastructure--apis) (/home/runner/work/filecoin-docs/filecoin-docs/docs/reference/README.md:25:7)

Executes with --fix to fix automatically

Checking failed

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! filecoin-docs@1.0.0 docs:links: `vuepress check-md docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the filecoin-docs@1.0.0 docs:links script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-09-18T13_19_33_151Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! filecoin-docs@1.0.0 docs:build: `npm run docs:links && vuepress build docs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the filecoin-docs@1.0.0 docs:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-09-18T13_19_33_168Z-debug.log
```

File: docs/mine/hardware-requirements.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ breadcrumb: 'Hardware requirements'
 
 The hardware requirements for Filecoin mining are tied to the computational resources needed to _seal_ a sector and generating regular _Proof of Spacetime_ for every sealed sector (_WindowPoSt_).
 
-These are computationally expensive operations which depend on the sector sizes used by the [Filecoin network](https://network.filecoin.io) on which the miner is running -nerpa, testnet, mainnet etc.-. For reference, the requirements listed below correspond to **32GiB sectors**, as used by mainnet and some testnets (_calibration_, _nerpa_).
+These are computationally expensive operations which depend on the sector sizes used by the [Filecoin network](https://network.filecoin.io) on which the miner is running -, testnet, mainnet etc.-. For reference, the requirements listed below correspond to **32GiB sectors**, as used by mainnet and some testnets (_calibration_).
```
Comment (johnnymatthews) [inline docs/mine/hardware-requirements.md:11]:
```suggestion
The Filecoin network requires storage providers to run computationally expensive operations. The cost of these operations depends on which network the storage provider is running; some testnets use a smaller sector size to increase the speeds of transactions artificially. For reference, the requirements listed below correspond to **32GiB sectors**, as used by mainnet and some testnets.
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Added two suggestions. Apart from those two, this is a good PR and ready to merge.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/get-started/lotus/chain.md`:
- Possibly missing comma found.
  ```
  ... Filecoin's blockchain grows relatively fast and a full sync will take a lon...
                                             ^^^^
  ```
- This phrase is duplicated. You should probably leave only “Current block”.
  ```
  ...t height State start height Lightweight Current block Current block - 2000 b...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...2000 blocks Current block - 2000 blocks Full chain Current block Genesis blo...
                                             ^^^^
  ```
- The verb “double-check” is spelled with a hyphen.
  ```
  ...rily, and may not be up-to-date. Please double check before using them. Cali...
                                             ^^^^^^^^^^^^
  ```
- A determiner may be missing.
  ```
  ...efore using them. Calibration: Download latest snapshot: lotus_cali_snapshot...
                                             ^^^^^^
  ```
- Do not mix variants of the same word (‘synch’ and ‘sync’) within a single text.
  ```
  ...ways to check your Lotus daemon's chain synching progress. # Sync status Use...
                                             ^^^^^^^^
  ```
- ‘wait’ (anticipate or expect something) seems less likely than ‘weight’ (mass).
  ```
  ...rrent state of your local chain: # Sync wait Use sync wait to output the sta...
                                             ^^^^
  ```
- Did you mean “to back up”?
  ```
  ...ith how many state roots you would like to backup: To create a pruned snapsh...
                                             ^^^^^^^^^
  ```
### `docs/get-started/lotus/installation.md`:
- The following link returned HTTP status code 302 (Found):
  https://github.com/filecoin-project/lotus/releases/latest
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://golang.org/dl/
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ing Amazon Web Services, are covered in Running in the cloud. # Minimal requ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  .... # Minimal requirements To run a Lotus node your computer must have: macOS ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...und 33Gib.  The full history was around 10TiB in June of 2021. WARNING These...
                                             ^^^^^
  ```
- When specifying a month and year, ‘of’ is unnecessary.
  ```
  ....  The full history was around 10TiB in June of 2021. WARNING These are the ...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...cal/go/bin to your path. For most Linux distributions you can run something ...
                                             ^^^^^^^^^^^^^
  ```
- A word may be missing after ‘a’.
  ```
  ...xport additional environment variables: a. If you have an AMD Zen or Intel I...
                                             ^^
  ```
- A word may be missing after ‘a’.
  ```
  ...adding these two environment variables: a. Some older Intel and AMD processo...
                                             ^^
  ```
- A word may be missing after ‘a’.
  ```
  ...dd the CGO_CFLAGS environment variable: a. By default, a 'multicore-sdr' opt...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...a 'multicore-sdr' option is used in the proofs library.  This feature is als...
                                             ^^^^^^
  ```
- The operating system from Apple is written “macOS”.
  ```
  ...efault and not visible in journalctl. # MacOS These instructions are specifi...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...by building from source. # Install with Homebrew You can quickly install Lot...
                                             ^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...sitory:  Run git checkout <RELEASE TAG> to checkout to the latest Lotus rele...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...GitHub release page  (opens new window) to checkout to that specific release...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ... that specific release. TIP If you want to checkout to a network other than ...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... other than mainnet, take a look at the Switching networks guide → Create ne...
                                             ^^^^^^^^^
  ```
- Possibly missing article found.
  ```
  ...t the Switching networks guide → Create necessary environment variable to al...
                                             ^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...sitory:  Run git checkout <RELEASE TAG> to checkout to the latest Lotus rele...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...GitHub release page  (opens new window) to checkout to that specific release...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ... that specific release. TIP If you want to checkout to a network other than ...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... other than mainnet, take a look at the Switching networks guide → If you ar...
                                             ^^^^^^^^^
  ```
- Unpaired symbol: ‘"’ seems to be missing
  ```
  ... at some tips for running Lotus in China". Some older Intel and AMD processo...
                                             ^
  ```
- The phrase ‘lots of’ might be wordy. Consider using “many”.
  ```
  ... chain. The daemon will start producing lots of log messages right away. Fro...
                                             ^^^^^^^
  ```
### `docs/get-started/lotus/switch-networks.md`:
- Possibly missing comma found.
  ```
  ...ue on this network. Calibnet make clean calibnet A test network with a minim...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ing lotus daemon: To change your $LOTUS_PATH run: export LOTUS_PATH=~/.new-l...
                                             ^^^^
  ```
- Did you mean “to back up”?
  ```
  ...th. # Backing up Lotus data If you wish to backup Lotus data, copy the ~/.lo...
                                             ^^^^^^^^^
  ```
### `docs/mine/hardware-requirements.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/filecoin-project/bellman
- Insert a space between the numerical value and the unit symbol.
  ```
  ...requirements listed below correspond to 32GiB sectors, as used by mainnet an...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... affected by slow disks. For example, a 32GiB expands to ~480GiB during the ...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...disks. For example, a 32GiB expands to ~480GiB during the sealing process. F...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...s. Filecoin network parameters are over 100GiB and need to be read and verif...
                                             ^^^^^^
  ```
- Use a comma after an introductory phrase.
  ```
  ...dressed with a fast swap drive or file. For this reasons, a minimal amount o...
                                             ^^^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  .... For this reasons, a minimal amount of 1TiB NVMe-based disk space for cache...
                                             ^^^^
  ```
- Possibly missing article found.
  ```
  ... cache Filecoin parameters and serve as general temporal storage location. A...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...have to perform operations of different nature which differ in how they use ...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ase 1 Yes (1 core or 1 core-complex) No 128GiB PoRep SDR encoding. Not amena...
                                             ^^^^^^
  ```
- Consider inserting a comma after the opening of your message.
  ```
  ..._USE_MULTICORE_SDR  (opens new window). Also see the Lotus seal workers guid...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...hase 2 Yes (when no GPU, all cores) Yes 128GiB Merkle tree generation using ...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...se 2 Yes (when no GPU, all cores) Yes ~ 192GiB Slow with just CPUs. Unsealin...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...th just CPUs. Unsealing Yes (1 core) No 128GiB  Proving WindowPoSt Yes (all ...
                                             ^^^^^^
  ```
- Possibly missing comma found.
  ```
  ..., the more CPU cores the faster Proving WinningPoSt Yes No - WinningPoSt is ...
                                             ^^^^^^^^^^^
  ```
- After the determiner ‘a’, noun phrases expressing time tend to have singular nouns.
  ```
  ... computation. Must be completed in a 25 seconds window. Note that the Lotus ...
                                             ^^^^^^^
  ```
- Did you mean “configuring”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ...te that the Lotus implementation allows to configure and delegate specific s...
                                             ^^^^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/get-started/lotus/chain.md`:
- Possibly missing comma found.
  ```
  ... Filecoin's blockchain grows relatively fast and a full sync will take a lon...
                                             ^^^^
  ```
- This phrase is duplicated. You should probably leave only “Current block”.
  ```
  ...t height State start height Lightweight Current block Current block - 2000 b...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...2000 blocks Current block - 2000 blocks Full chain Current block Genesis blo...
                                             ^^^^
  ```
- The verb “double-check” is spelled with a hyphen.
  ```
  ...rily, and may not be up-to-date. Please double check before using them. Cali...
                                             ^^^^^^^^^^^^
  ```
- A determiner may be missing.
  ```
  ...efore using them. Calibration: Download latest snapshot: lotus_cali_snapshot...
                                             ^^^^^^
  ```
- Do not mix variants of the same word (‘synch’ and ‘sync’) within a single text.
  ```
  ...ways to check your Lotus daemon's chain synching progress. # Sync status Use...
                                             ^^^^^^^^
  ```
- ‘wait’ (anticipate or expect something) seems less likely than ‘weight’ (mass).
  ```
  ...rrent state of your local chain: # Sync wait Use sync wait to output the sta...
                                             ^^^^
  ```
- Did you mean “to back up”?
  ```
  ...ith how many state roots you would like to backup: To create a pruned snapsh...
                                             ^^^^^^^^^
  ```
### `docs/get-started/lotus/installation.md`:
- The following link returned HTTP status code 302 (Found):
  https://github.com/filecoin-project/lotus/releases/latest
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://golang.org/dl/
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...ing Amazon Web Services, are covered in Running in the cloud. # Minimal requ...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  .... # Minimal requirements To run a Lotus node your computer must have: macOS ...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...und 33Gib.  The full history was around 10TiB in June of 2021. WARNING These...
                                             ^^^^^
  ```
- When specifying a month and year, ‘of’ is unnecessary.
  ```
  ....  The full history was around 10TiB in June of 2021. WARNING These are the ...
                                             ^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...cal/go/bin to your path. For most Linux distributions you can run something ...
                                             ^^^^^^^^^^^^^
  ```
- A word may be missing after ‘a’.
  ```
  ...xport additional environment variables: a. If you have an AMD Zen or Intel I...
                                             ^^
  ```
- A word may be missing after ‘a’.
  ```
  ...adding these two environment variables: a. Some older Intel and AMD processo...
                                             ^^
  ```
- A word may be missing after ‘a’.
  ```
  ...dd the CGO_CFLAGS environment variable: a. By default, a 'multicore-sdr' opt...
                                             ^^
  ```
- An apostrophe may be missing.
  ```
  ...a 'multicore-sdr' option is used in the proofs library.  This feature is als...
                                             ^^^^^^
  ```
- The operating system from Apple is written “macOS”.
  ```
  ...efault and not visible in journalctl. # MacOS These instructions are specifi...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...by building from source. # Install with Homebrew You can quickly install Lot...
                                             ^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...sitory:  Run git checkout <RELEASE TAG> to checkout to the latest Lotus rele...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...GitHub release page  (opens new window) to checkout to that specific release...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ... that specific release. TIP If you want to checkout to a network other than ...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... other than mainnet, take a look at the Switching networks guide → Create ne...
                                             ^^^^^^^^^
  ```
- Possibly missing article found.
  ```
  ...t the Switching networks guide → Create necessary environment variable to al...
                                             ^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...sitory:  Run git checkout <RELEASE TAG> to checkout to the latest Lotus rele...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ...GitHub release page  (opens new window) to checkout to that specific release...
                                             ^^^^^^^^^^^
  ```
- The verb ‘check out’ is spelled as two words. The noun ‘checkout’ is spelled as one.
  ```
  ... that specific release. TIP If you want to checkout to a network other than ...
                                             ^^^^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... other than mainnet, take a look at the Switching networks guide → If you ar...
                                             ^^^^^^^^^
  ```
- Unpaired symbol: ‘"’ seems to be missing
  ```
  ... at some tips for running Lotus in China". Some older Intel and AMD processo...
                                             ^
  ```
- The phrase ‘lots of’ might be wordy. Consider using “many”.
  ```
  ... chain. The daemon will start producing lots of log messages right away. Fro...
                                             ^^^^^^^
  ```
### `docs/get-started/lotus/switch-networks.md`:
- Possibly missing comma found.
  ```
  ...ue on this network. Calibnet make clean calibnet A test network with a minim...
                                             ^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...ing lotus daemon: To change your $LOTUS_PATH run: export LOTUS_PATH=~/.new-l...
                                             ^^^^
  ```
- Did you mean “to back up”?
  ```
  ...th. # Backing up Lotus data If you wish to backup Lotus data, copy the ~/.lo...
                                             ^^^^^^^^^
  ```
### `docs/mine/hardware-requirements.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://github.com/filecoin-project/bellman
- Insert a space between the numerical value and the unit symbol.
  ```
  ...requirements listed below correspond to 32GiB sectors, as used by mainnet an...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ... affected by slow disks. For example, a 32GiB expands to ~480GiB during the ...
                                             ^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...disks. For example, a 32GiB expands to ~480GiB during the sealing process. F...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...s. Filecoin network parameters are over 100GiB and need to be read and verif...
                                             ^^^^^^
  ```
- Use a comma after an introductory phrase.
  ```
  ...dressed with a fast swap drive or file. For this reasons, a minimal amount o...
                                             ^^^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  .... For this reasons, a minimal amount of 1TiB NVMe-based disk space for cache...
                                             ^^^^
  ```
- Possibly missing article found.
  ```
  ... cache Filecoin parameters and serve as general temporal storage location. A...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...have to perform operations of different nature which differ in how they use ...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...ase 1 Yes (1 core or 1 core-complex) No 128GiB PoRep SDR encoding. Not amena...
                                             ^^^^^^
  ```
- Consider inserting a comma after the opening of your message.
  ```
  ..._USE_MULTICORE_SDR  (opens new window). Also see the Lotus seal workers guid...
                                             ^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...hase 2 Yes (when no GPU, all cores) Yes 128GiB Merkle tree generation using ...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...se 2 Yes (when no GPU, all cores) Yes ~ 192GiB Slow with just CPUs. Unsealin...
                                             ^^^^^^
  ```
- Insert a space between the numerical value and the unit symbol.
  ```
  ...th just CPUs. Unsealing Yes (1 core) No 128GiB  Proving WindowPoSt Yes (all ...
                                             ^^^^^^
  ```
- Possibly missing comma found.
  ```
  ..., the more CPU cores the faster Proving WinningPoSt Yes No - WinningPoSt is ...
                                             ^^^^^^^^^^^
  ```
- After the determiner ‘a’, noun phrases expressing time tend to have singular nouns.
  ```
  ... computation. Must be completed in a 25 seconds window. Note that the Lotus ...
                                             ^^^^^^^
  ```
- Did you mean “configuring”? Or maybe you should add a pronoun? In active voice, ‘allow’ + ‘to’ takes an object, usually a pronoun.
  ```
  ...te that the Lotus implementation allows to configure and delegate specific s...
                                             ^^^^^^^^^^^^
  ```

**Review by johnnymatthews — APPROVED** [review]

**Pull request merged**